apis/sgv2-restapi/src/test/java/io/stargate/sgv2/it/RestApiV2QIntegrationTestBase.java

```diff
@@ -151,11 +152,11 @@ public void cleanupPerClass() {
    */
 
   public String testKeyspaceName() {
-    return testKeyspaceName;
+    return Objects.requireNonNull(testKeyspaceName, "'testKeyspaceName' not initialized");
```

Comment:
Added because running of individual test from IDE for classes that only use per-class keyspaces will otherwise result in weird query execution failure ("No keyspace 'null' found").